sys/kern/kern_malloc.c

```diff
@@ -97,7 +99,7 @@ __FBSDID("$FreeBSD$");
 #endif
 
 #if __has_feature(capabilities)
-#include <cheri/cherireg.h>
+#include <machine/cherireg.h>
```

Comment:
This should be automatically pulled in by cheric.h, no?

Reply:
Yes, will drop it.